calico-cloud_versioned_docs/version-3.15/variables.js

```diff
@@ -12,11 +12,11 @@ const variables = {
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.15.1-1',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.15.1-1',
+  cloudversion: 'v3.15.1-8',
+  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.15.1-8',
   clouddownloadbase: 'https://installer.calicocloud.io',
   cloudoperatorimage: 'us-docker.pkg.dev/tigera-prod-01/calico-cloud-public/cc-operator',
-  imageassuranceversion: 'v1.3.4',
+  imageassuranceversion: 'v1.4.1',
```

Comment:
@tmjd v1.4.1 is the correct version for the CLI and admission controller. I'm assuming that's what this parameter is used for. 

Other pieces of IA were left on v1.4.0, but that should be fine.

Reply:
Yes the v1.4.1 is only for the user to verify the version of the CLI.
The version of Admission controller comes from the `cloudversion` that was also updated with this PR.